games-arcade/freegish/freegish-1.0.recipe

```diff
@@ -0,0 +1,57 @@
+SUMMARY="A free version of the game Gish"
+DESCRIPTION="FreeGish is an open-source, freely redistributable version of Gish, a \
+physics-based platforming game featuring a ball of tar."
+HOMEPAGE="http://github.com/freegish/freegish"
+COPYRIGHT="2012-2014 Joris van de Donk"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/freegish/freegish/archive/f53d9153.zip"
+CHECKSUM_SHA256="2d5e0f9d24ddc648e4cdf45adf50043ec13ecd991da61166d670f4be74447665"
+PATCHES="freegish-1.0.patchset"
+
+ARCHITECTURES="?x86 x86_gcc2 ?x86_64"
+
+PROVIDES="
+	app:FreeGish = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libogg
+	lib:libopenal
+	lib:libpng
+	lib:libsdl
+	lib:libvorbis
+	lib:libz
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libogg
+	devel:libopenal
+	devel:libpng
+	devel:libsdl
+	devel:libvorbis
+	devel:libz
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:make
+	cmd:gcc
+"
+
+BUILD()
+{
+	mkdir -p build && cd build
+	cmake .. -DDATAPATH='"$appsDir"'
+	make
+}
+
+INSTALL()
+{
+	cd ..
+	mkdir -p $appsDir/FreeGish
+	cp -r  freegish-1.0 $appsDir/FreeGish
+	mv $appsDir/FreeGish/freegish $appsDir/FreeGish/FreeGish
+	addAppDeskbarSymlink $appsDir/FreeGish/FreeGish "FreeGish"
```

Comment:
The last part is the link name and is optional. And since it matches the binary name already it is superfluous